extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -234,8 +234,11 @@ export class ConsentUI {
     const {classList} = this.parent_;
     classList.add('amp-active');
     classList.remove('amp-hidden');
-    // Add to fixed layer
-    this.baseInstance_.getViewport().addToFixedLayer(this.parent_);
+
+    this.baseInstance_
+      .getViewport()
+      .addToFixedLayer(this.parent_, /* forceTransfer */ true);
```

Comment:
Possible to add a test?

Reply:
This is not easy to reproduce with tests. Because its a fix for an unsupported version of safari.

Reply:
We can verify that the right `FixedLayer` call is made, but an e2e would be too contrived.